lib/aur-sync

```diff
@@ -118,6 +118,8 @@ tmp=$(mktemp -dt "$argv0".XXXXXXXX)
 tmp_view=$(mktemp -dt view.XXXXXXXX)
 trap 'trap_exit' EXIT
 
+mkdir -p "$tmp_view/diff" "$tmp_view/new" "$tmp_view/pkg"
```

Comment:
More compact: `mkdir -p "$tmp_view"/{diff,new,pkg}`

I've thought on some other names: 
* diff__s__ (compare `refs` in .git, and indeed multiple `.diff` files are stored there)
* queue (the files symlinked there reflect the build queue)
  * possible confusion when reading the code with `"$tmp"/queue` which is a regular file...
 
Since `new` is not yet implemented perhaps we can leave the reference (and according name) for a different PR.

Reply:
I like the `diffs`, not so keep on `queue`, maybe `pkgs` (also plural?), will remove `new`